_health-care/_js/utils/validations.js

```diff
@@ -17,8 +17,13 @@ function isValidPhone(value) {
   return /^\d{3}-\d{3}-\d{4}$/.test(value);
 }
 
+function isValidEmail(value) {
+  return /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/.test(value);
```

Comment:
Yeah, email addresses are really hard to validate... I found this one on StackOverflow, which I put here in to start with, but not committed to it. I also looked into the validator.js library and their validation logic is even more complex. How specific do we want to be with validating emails? Is this sufficient: some number of characters + @ + some number of characters + . + 2 or more alphabetical characters?